modules/telestion-example/src/main/java/de/wuespace/telestion/example/information/SimpleInformation.java

```diff
@@ -0,0 +1,20 @@
+package de.wuespace.telestion.example.information;
+
+import de.wuespace.telestion.api.message.header.Information;
+import de.wuespace.telestion.api.message.header.serialization.SerializationInfo;
+import io.vertx.core.MultiMap;
+
+public record SimpleInformation(
+		@SerializationInfo(name = "value1", defaultValue = "4") int value1,
+		@SerializationInfo(name = "value2") Integer value2,
+		@SerializationInfo(name = "char1") char char1,
+		@SerializationInfo(name = "char2") Character char2,
+		@SerializationInfo(name = "str") String str,
+		@SerializationInfo(name = "unsigned-int", defaultValue = "0", isUnsigned = true) int unsignedInt,
+		@SerializationInfo(name = "boolean") boolean bool
+) implements Information {
+	// fromHeaders(MultiMap) is optional but improves the developer experience in our opinion
```

Comment:
Please add `// Note YourName: ...` so that we can ask you afterwards.

Reply:
@fussel178 it was your idea, want to annotate it with your name alone, or do you want us both to be "responsible" for this comment? :)